Compilation error on Windows

```
error[E0603]: trait `Downloader` is private
  --> src\commands\app\download.rs:6:13
   |
6  |     cache::{Downloader, Handle},
   |             ^^^^^^^^^^ private trait
   |
note: the trait `Downloader` is defined here
  --> C:\Users\ךינשגכהד\.cargo\git\checkouts\sprinkles-1ae7f229d9e11ff9\b4fc778\src\cache.rs:14:5
   |
14 | use crate::packages::downloading::Downloader;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: import `Downloader` directly
   |
6  |     cache::{sprinkles::packages::downloading::Downloader, Handle},
   |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0609]: no field `url` on type `InstallConfig`
   --> src\commands\virustotal.rs:179:26
    |
179 |                         .url
    |                          ^^^ unknown field
    |
help: a field with a similar name exists
    |
179 |                         .urls
    |                          ~~~~

warning: use of deprecated method `ratatui::Frame::<'_>::size`: use .area() as its the more correct name
   --> src\commands\credits.rs:362:19
    |
362 |             frame.size(),
    |                   ^^^^
    |
    = note: `#[warn(deprecated)]` on by default

Some errors have detailed explanations: E0603, E0609.
For more information about an error, try `rustc --explain E0603`.
warning: `sfsu` (bin "sfsu") generated 1 warning
error: could not compile `sfsu` (bin "sfsu") due to 2 previous errors; 1 warning emitted
error: failed to compile `sfsu v1.14.0 (https://github.com/winpax/sfsu#a98d7b05)`, intermediate artifacts can be found at `C:\Users\0BAC~1\AppData\Local\Temp\cargo-installACIsqd`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

--- a/src/commands/app/download.rs
+++ b/src/commands/app/download.rs
@@ -3,9 +3,9 @@ use std::time::Duration;
 use clap::Parser;
 
 use sprinkles::{
-    cache::{Downloader, Handle},
+    cache::{DownloadHandle, Handle},
     contexts::ScoopContext,
-    packages::reference::package,
+    packages::{downloading::Downloader, reference::package},
     progress::indicatif::{MultiProgress, ProgressBar},
     requests::AsyncClient,
     Architecture,
@@ -48,7 +48,7 @@ impl super::Command for Args {
         let pb = ProgressBar::new_spinner().with_message("Initializing download(s)");
         pb.enable_steady_tick(Duration::from_millis(100));
 
-        let downloaders: Vec<Downloader> =
+        let downloaders: Vec<DownloadHandle> =
             futures::future::try_join_all(self.packages.into_iter().map(|package| {
                 let mp = mp.clone();
                 async move {
@@ -62,7 +62,7 @@ impl super::Command for Args {
                     let downloaders = dl.into_iter().map(|dl| {
                         let mp = mp.clone();
                         async move {
-                            match Downloader::new::<AsyncClient>(dl, Some(&mp)).await {
+                            match DownloadHandle::new::<AsyncClient>(dl, Some(&mp)).await {
                                 Ok(dl) => anyhow::Ok(dl),
                                 Err(e) => match e {
                                     sprinkles::cache::Error::ErrorCode(status) => {

--- a/src/commands/virustotal.rs
+++ b/src/commands/virustotal.rs
@@ -176,7 +176,7 @@ impl super::Command for Args {
                 } else {
                     manifest
                         .install_config(self.arch)
-                        .url
+                        .urls
                         .map(|url| url.map(SearchType::Url).to_vec())
                 };
 

